ui-kit/src/main/kotlin/org/p2p/uikit/atoms/skeleton/UiKitSkeletonLineView.kt

```diff
@@ -0,0 +1,36 @@
+package org.p2p.uikit.atoms.skeleton
+
+import androidx.core.content.ContextCompat
+import androidx.core.view.isVisible
+import android.content.Context
+import android.util.AttributeSet
+import com.facebook.shimmer.Shimmer
+import com.facebook.shimmer.ShimmerFrameLayout
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.AtomSkeletonLineViewBinding
+import org.p2p.uikit.utils.inflateViewBinding
+
+class UiKitSkeletonLineView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : ShimmerFrameLayout(context, attrs, defStyleAttr) {
+
+    private val binding = inflateViewBinding<AtomSkeletonLineViewBinding>()
```

Comment:
we didn't agree on naming for views that start with Design views naming. No `atoms` `molecules` or other stuff is used for naming. 
widget_skeleton_line would be ok